src/Hyperion/Main.hs

```diff
@@ -169,7 +195,10 @@ defaultMainWith presetConfig bks = do
         Version -> putStrLn $ "Hyperion " <> showVersion version
         List -> doList bks
         Run -> do _ <- doRun bks; return ()
-        Analyze -> doAnalyze config bks
+        Analyze -> doAnalyze config (pack packageName) bks
 
-defaultMain :: [Benchmark] -> IO ()
+defaultMain
+  :: String -- ^ Packahe name, user provided.
```

Comment:
Also please keep :: on the same row as function name, this improve experience when you grep for function definition, just search `function ::`